src/styles/foundation/_focus-ring.scss

```diff
@@ -8,15 +8,11 @@
 ///
 
 @mixin focus-ring($size: 'base', $border-width: 0, $style: 'base') {
-  $stroke: rem(2px);
+  $stroke: 2px;
   // calc does not like performing addition with a unitless number (`0`, NOT `0px`)
-  // This is a problem because `rem(0px)` returns `0`, not `0px`.
+  // This is a problem because `0px` returns `0`, not `0px`.
   // Make sure that we can handle unitless zeros by not trying to do math with them
-  $offset: if(
-    $border-width == 0,
-    rem(1px),
-    calc(#{$border-width} + #{rem(1px)})
-  );
+  $offset: if($border-width == 0, 1px, calc(#{$border-width} + 1px));
```

Comment:
Our regex may have unintentionally updated this comment/variable. We should probably tophat this variable, make sure it works as expected, and update or remove the comment. Thoughts?